### ISSUE TITLE
Long Pause?

### ISSUE THREAD

**Issue by brony9tynine** [Roman971/OoT-Randomizer#60]
When I pause sometimes it takes a long time and then freezes requiring to change in/out of full screen. I am using retro-arch version 1.8.6 ( i think thats the version) with ParraLLEI

**Comment by Roman971** [comment]
Thanks for the report. This is most likely an emulator issue and not something directly caused by the randomizer, so I don't know if I'll be able to help, but I'll try to run tests soon to see if I can find a cause.

**Comment by brony9tynine** [comment]
yeah probably i dont have the specific randomized rom anymore and its not
doing it. i was basically using max rando (glitchless) if that helps
